1 Lesson/Homework/Garden.rb

```diff
@@ -0,0 +1,145 @@
+class Raspberry
+  @@states = [:Отсутствует, :Цветение, :Зеленая, :Красная]
+
+  def initialize(index)
+    @index = index
+    @state = @@states[0]
+  end
+
+  def grow!
+    if @state != :Красная
+      @state = @@states[(@@states.find_index(@state) + 1) % @@states.size]
+    end
+  end
+
+  def ripe?
+    @state == :Красная
+  end
+
+  def grab!
+    @state = @@states[0]
+  end
+end
+
+class RaspberryBush
+  attr_reader :berries_quantity
+
+  def initialize(berries_quantity)
+    @berries_quantity = berries_quantity
```

Comment:
Why you save it?

Reply:
> 

Used quantity variable for calculation of reward for Human object;
Might have implemented return of quantity inside grow_all! method though, so that there wouldn't be any need for attr_reader